README.md

```diff
@@ -105,6 +105,19 @@ Bevy is free and open source! All code in this repository is dual-licensed under
 
 at your option. This means you can select the license you prefer! This dual-licensing approach is the de-facto standard in the Rust ecosystem and there are [very good reasons](https://github.com/bevyengine/bevy/issues/2373) to include both.
 
-Unless you explicitly state otherwise, any contribution intentionally submitted
-for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any
-additional terms or conditions.
+Some small sections of the engine's code fall under an additional MIT-compatible open source license due to their external origins.
```

Comment:
```suggestion
Some of the engine's code carries additional copyright notices and license terms (due to their external origins).
These are generally BSD-like, but exact details vary by crate:
```
(We need to be thinking about `bevy_html_colours` here, once that gets made/cleared up)